### DISCUSSION TITLE
What's the point of emitting the `block.timestamp` in the create event?

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/flow#194] [Q&A] [answered]
Here: 

https://github.com/sablier-labs/flow/blob/76bcae36e8286b2465adcf8ab80d8e24d8548bb4/src/interfaces/ISablierFlow.sol#L40

The `lastUpdateTime` can be inferred from the block timestamp of the block in which the create event was emitted. AFAIK, this property is available in the subgraph.

Asking you @sablier-labs/solidity and @razgraf (to double-check my understanding of the subgraph and whether this is a problem)?

**Comment by smol-ninja** [comment]
Agree that we can remove it.

**Reply by PaulRBerg** [reply]
Great, thanks for confirming. Waiting for @razgraf to confirm that this is OK.

**Comment by razgraf** [comment, marked as answer]
We have access, by default, to block timestamp, number etc. in the transaction metadata provided by indexers/subgraphs, so from our POV there's no need to store it.

**Reply by PaulRBerg** [reply]
OK great, thanks for confirming.
